test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,181 @@
 
 describe("Plots", () => {
   describe("AreaPlot", () => {
-    // HACKHACK #1798: beforeEach being used below
-    it("renders correctly with no data", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
-      let plot = new Plottable.Plots.Area();
-      plot.x((d: any) => d.x, xScale);
-      plot.y((d) => d.y, yScale);
-      assert.doesNotThrow(() => plot.renderTo(svg), Error);
-      assert.strictEqual(plot.width(), 400, "was allocated width");
-      assert.strictEqual(plot.height(), 400, "was allocated height");
-      svg.remove();
-    });
+    describe("Basic Usage", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let plot: Plottable.Plots.Area<{}>;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+        plot = new Plottable.Plots.Area();
+        plot.x((d) => d.x, xScale);
+        plot.y((d) => d.y, yScale);
+      });
 
-    it("adds a padding exception to the y scale at the constant y0 value", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
-      yScale.padProportion(0.1);
-      let constantY0 = 30;
-      yScale.addIncludedValuesProvider((scale: Plottable.Scales.Linear) => [constantY0, constantY0 + 10]);
-      let plot = new Plottable.Plots.Area();
-      plot.x((d) => d.x, xScale);
-      plot.y((d) => d.y, yScale);
-      plot.y0(constantY0);
-      plot.addDataset(new Plottable.Dataset([{ x: 0, y: constantY0 + 5 }]));
-      plot.renderTo(svg);
-      assert.strictEqual(yScale.domain()[0], constantY0, "y Scale doesn't pad beyond 0 when used in a Plots.Area");
-      svg.remove();
-    });
-  });
+      it("renders correctly with no data", () => {
```

Comment:
Test name seems too general for me.  Instead can we say that it does not error upon rendering if no data is inputted?